xray/config.go

```diff
@@ -37,29 +37,28 @@ var globalCfg = newGlobalConfig()
 
 func newGlobalConfig() *globalConfig {
 
-	// Try to get the X-Ray daemon address from an environment variable
-	var daemonAddress *net.UDPAddr
-	var err error
-
-	environmentDaemonAddress := os.Getenv("AWS_XRAY_DAEMON_ADDRESS")
+	ret := &globalConfig{
```

Comment:
Here we should call `loadLogConfig("", "")` method. One is to get the default value, the other is to init log with default setting.